closed/autoconf/custom-hook.m4

```diff
@@ -295,6 +298,8 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
     OPENJ9_PLATFORM_CODE=xr32
     OPENJ9_BUILDSPEC=linux_arm_linaro
     OPENJ9_LIBS_SUBDIR=default
+  elif test "x$OPENJ9_CPU" = xaarch64 ; then
+    OPENJ9_PLATFORM_CODE=xr64
```

Comment:
This will require the addition of `PLATFORM-xr64` to the set of `platforms` in `openj9/jcl/jpp_configuration.xml`.

Reply:
Thank you for pointing that out.  I added it in OpenJ9 PR 4350.

Reply:
FYI, you can create a link to that PR with the text `eclipse/openj9#4350` to get eclipse/openj9#4350.